src/lib/utils/html.ts

```
@@ -0,0 +1,56 @@
+      a: ['href'],
+      b: [],
+      em: [],
+      i: [],
+      strong: [],
+    },
+  };
+
+  if (options.allowedTags) {
+    defaultOptions.whiteList = options.allowedTags.reduce(
+      (array, tag) => {
+        array[tag] = (defaultOptions.whiteList && defaultOptions.whiteList[tag]) || [];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)